modules/sitemaps/sitemaps.php

```diff
@@ -250,7 +257,14 @@ function jetpack_print_news_sitemap_xsl() {
 function jetpack_print_sitemap() {
 	global $wpdb, $post;
 
-	$xml = get_transient( 'jetpack_sitemap' );
+	$xml = get_transient(
+		/**
+		 * Filter transient name.
+		 *
```

Comment:
Hi, 

Sorry did not understand, do you want me to add new var? 

Warm regards